cptoml.py

```diff
@@ -118,7 +129,7 @@ def _linemake(key, value, comment=None) -> str:
         del result, key, value, comment
         raise TypeError("Unsupported type for toml")
     if comment is not None:
-        result += " # " + comment
+        result += f" # {comment}"
```

Comment:
Slower, and allocates ram temporarily.

F strings are cool, but allocate a bunch of ram (which gc will have to lag the board to clean).
So unless you are doing 2 or more `+` it ain't worth it.

Reply:
Ok, noted.